torch/fx/graph.py

```diff
@@ -767,14 +767,14 @@ def gen_fn_def(self, free_vars, maybe_return_annotation):
                 self.pytree_info.in_spec.children_specs[0].type == tuple and \
                 self.pytree_info.in_spec.children_specs[1].type == dict
             fn_kwargs = '{}'
-            fn_signature = f"[{', '.join(fn_args)}], self._in_spec"
+            fn_signature = f"({', '.join(fn_args)},), self._in_spec"
             if has_args_kwargs_tuple:
                 count_args = self.pytree_info.in_spec.children_specs[0].num_children
                 fn_args = self.pytree_info.orig_args[:count_args]
                 fn_kwargs = '{' + ', '.join(f"'{k}':{v}" for k, v in zip(
                                   self.pytree_info.in_spec.children_specs[1].context,
                                   self.pytree_info.orig_args[count_args:])) + '}'
-                fn_signature = f"([{', '.join(fn_args)}], {fn_kwargs}), self._in_spec"
+                fn_signature = f"(({', '.join(fn_args)},), {fn_kwargs}), self._in_spec"
```

Comment:
```suggestion
                fn_signature = f"(({tuple(fn_args)},), {fn_kwargs}), self._in_spec"
```
? No need to reinvent the tuple printer?

Reply:
`fn_args` is `list[str]`, so `tuple(fn_args)` will get:

```python
('x', 'y')
```

but the expected signature is:

```python
(x, y,)
```